run/jobs/test/system.test.js

```diff
@@ -69,9 +69,13 @@ describe('End-to-End Tests', () => {
       `resource.labels.job_name = "${SERVICE_NAME}" ` +
       `resource.labels.location = "${REGION}" ` +
       `timestamp>="${dateMinutesAgo(new Date(), 5)}"`;
+    console.log(preparedFilter);
+
+    const sleep = ms => new Promise(r => setTimeout(r, ms));
```

Comment:
We import `delay` module in line 16 and might as well use it.  Or, alternatively, stop importing `delay` module and just use this `sleep` everywhere (and I would personally prefer the latter).